### PR TITLE
add 2.1 dockerfiles and tensorflow artifacts

### PR DESCRIPTION
**Pull request by Elizaaaaa**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/2.1/Dockerfile.gpu [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ && rm -rf /var/lib/apt/lists/*
+
+# The 'apt-get install' of nvinfer-runtime-trt-repo-ubuntu1804-4.0.1-ga-cuda10.0
+# adds a new list which contains libnvinfer library, so it needs another
+# 'apt-get update' to retrieve that list before it can actually install the
+# library.
+# We don't install libnvinfer-dev since we don't need to build against TensorRT,
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
```
Comment (saimidu) [inline docker/2.1/Dockerfile.gpu]: Please add justification in a comment for why this particular library is version 5.0.2 and cuda 10.0, while the rest of the libraries are for cuda 10.1.

File: docker/2.1/Dockerfile.gpu [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+ && apt-get update \
+ && apt-get -y install --no-install-recommends \
+    nginx \
+    nginx-module-njs \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install -U --no-cache-dir \
+    awscli==1.17.11 \
```
Comment (saimidu) [inline docker/2.1/Dockerfile.gpu]: Please unpin awscli, as it seems to be changing too quickly for a stable build with a version pin.

File: docker/2.1/Dockerfile.gpu [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends nvinfer-runtime-trt-repo-ubuntu1804-5.0.2-ga-cuda10.0 \
+ && apt-get update \
+ && apt-get install -y --no-install-recommends libnvinfer6=${TF_TENSORRT_VERSION}-1+cuda10.1 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* 
+ #&& rm /usr/lib/x86_64-linux-gnu/libnvinfer_plugin* \
+ #&& rm /usr/lib/x86_64-linux-gnu/libnvcaffe_parser* \
+ #&& rm /usr/lib/x86_64-linux-gnu/libnvparsers*
```
Comment (saimidu) [inline docker/2.1/Dockerfile.gpu]: Do these no longer need to be deleted? Is this meant to be added in later on, or is it dead code?

File: docker/2.1/Dockerfile.gpu [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+RUN curl $TFS_URL -o /usr/bin/tensorflow_model_server \
+ && chmod 555 /usr/bin/tensorflow_model_server
+
+# put tensorflow library (with only error_codes) to python dist-packages
+WORKDIR /
+RUN cd /usr/local/lib/python3.*/dist-packages/ && \
+    mv /sagemaker/tensorflow-2.1 ./tensorflow && \
+    # Delete the remaining tensorflow folder to avoid confusing python import
+    rm -rf /sagemaker/tensorflow
```
Comment (saimidu) [inline docker/2.1/Dockerfile.gpu]: Please re-format these lines as per the recommended format for multi-line RUN statements (as used in the other RUN statements in this dockerfile).

File: docker/2.1/Dockerfile.cpu [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    python3-pip \
+    python3-setuptools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN ${PIP} --no-cache-dir install --upgrade pip setuptools
+
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install --no-cache-dir \
+    awscli==1.17.11 \
```
Comment (saimidu) [inline docker/2.1/Dockerfile.cpu]: Please unpin awscli, as it seems to be changing too quickly for a stable build with a version pin.

File: docker/2.1/Dockerfile.cpu [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+    tensorflow-serving-api==2.1.0
+
+COPY ./sagemaker /sagemaker
+
+# put tensorflow library (with only error_codes) to python dist-packages
+WORKDIR /
+RUN cd /usr/local/lib/python3.*/dist-packages/ && \
+    mv /sagemaker/tensorflow-2.1 ./tensorflow && \
+    # Delete the remaining tensorflow folder to avoid confusing python import
+    rm -rf /sagemaker/tensorflow
```
Comment (saimidu) [inline docker/2.1/Dockerfile.cpu]: Please re-format these lines as per the recommended format for multi-line RUN statements (as used in the other RUN statements in this dockerfile).

Comment (Elizaaaaa) [reply docker/2.1/Dockerfile.cpu]: update all the reviews above

**Pull request merged**